### ISSUE TITLE
us-api.i.posthog.com Error while using PraisonAI (--agents)

### ISSUE THREAD

**Issue by bamit99** [danielmiessler/Fabric#414]
I am getting the errors below when using the --agents switch with fabric. I think this is probably because the tool is reaching some sort of limit while doing Internet Search or similar. How to avoid it? The tool comes back with the answers but I am guessing with limited information since we run into limitations. I am using Deepseek v2 for OpenAI and the usual Google Keys for YT and Google API,

2024-05-23 12:07:37,907 - 139747531929152 - _common.py-_common:120 - ERROR: Giving up send_request(...) after 4 tries (requests.exceptions.ConnectionError: HTTPSConnectionPool(host='us-api.i.posthog.com', port=443): Max retries exceeded with url: /batch/ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f198c61d550>: Failed to establish a new connection: [Errno 111] Connection refused')))
2024-05-23 12:07:38,170 - 139747020236352 - _common.py-_common:120 - ERROR: Giving up send_request(...) after 4 tries (requests.exceptions.ConnectionError: HTTPSConnectionPool(host='us-api.i.posthog.com', port=443): Max retries exceeded with url: /batch/ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f198c563c10>: Failed to establish a new connection: [Errno 111] Connection refused')))
2024-05-23 12:07:38,841 - 139746995058240 - _common.py-_common:120 - ERROR: Giving up send_request(...) after 4 tries (requests.exceptions.ConnectionError: HTTPSConnectionPool(host='us-api.i.posthog.com', port=443): Max retries exceeded with url: /batch/ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f198c503d50>: Failed to establish a new connection: [Errno 111] Connection refused')))
2024-05-23 12:07:39,284 - 139747011843648 - _common.py-_common:120 - ERROR: Giving up send_request(...) after 4 tries (requests.exceptions.ConnectionError: HTTPSConnectionPool(host='us-api.i.posthog.com', port=443): Max retries exceeded with url: /batch/ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f198c502250>: Failed to establish a new connection: [Errno 111] Connection refused'))

**Comment by bamit99** [comment]
Yes. Fabric folder shows a test.yaml file that has all the PraisonAI agents needed for the request to work.

**Comment by eous** [comment]
You can set the environment variable EC_TELEMETRY to false or some other non true boolean value to disable the anonymous embed chain telemetry collection.

**Comment by bamit99** [comment]
Let me try and get back to you.

**Comment by julliannailluj** [comment]
> You can set the environment variable EC_TELEMETRY to false or some other non true boolean value to disable the anonymous embed chain telemetry collection.

Canyou tell me where is that EC_TELEMETRY located? Thanks

**Comment by bamit99** [comment]
I tried creating a System Env with a value of False but it failed. Same error messages.

**Comment by eous** [comment]
https://github.com/embedchain/embedchain/blob/9544c69c55618b9ff0e3d413b9a246647a0338c3/embedchain/telemetry/posthog.py#L21

**Comment by ksylvan** [comment]
Old issue. Closing this.
